genesis/src/main.rs

```diff
@@ -301,6 +313,10 @@ fn main() -> Result<(), Box<dyn error::Error>> {
                     1,
                 ),
             ),
+            (
+                storage_mining_keypair.pubkey(),
+                storage_contract::create_mining_pool_account(100_000_000),
```

Comment:
Should we pass number of lamports as command line argument?

Reply:
Yeah i guess I should.